### PR TITLE
Fix read avatars overflow from the right chat panel with a maximized widget on bubble message layout

### PR DESCRIPTION
**Pull request by luixxiul**
Address one of the issues on https://github.com/vector-im/element-web/issues/21774

This PR fixes the issue that read avatars (read receipts) overflow from the right chat panel with a maximized widget on bubble message layout. The issue is neither reproduced on modern layout nor IRC layout.

Before:
before

After:
after

There can be a couple of edge cases which are not often reproduced, but most cases are addressed, such as avatars next to a message, an event line, and also an event list summary.

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: defect

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix read avatars overflow from the right chat panel with a maximized widget on bubble message layout ([\#8470](https://github.com/matrix-org/matrix-react-sdk/pull/8470)). Contributed by @luixxiul.

### CODE REVIEW

File: res/css/views/right_panel/_TimelineCard.scss [hunk 1]
```diff
@@ -90,7 +90,8 @@ limitations under the License.
         }
```
Comment (MadLittleMods) [inline res/css/views/right_panel/_TimelineCard.scss:90]: Let's add a before/after screenshot of the same area to the issue description for easy comparison.

Comment (luixxiul) [reply res/css/views/right_panel/_TimelineCard.scss:90]: ok, please wait for a moment..

File: res/css/views/right_panel/_TimelineCard.scss [hunk 2]
```diff
@@ -141,11 +142,19 @@ limitations under the License.
...
     }
 
-    &.mx_BaseCard {
-        // For a chat timeline on the right panel when the widget is maximised
-        // TODO: rename ThreadPanel
-        &.mx_ThreadPanel {
-            padding-right: 8px; // .mx_RightPanel padding
+    .mx_GenericEventListSummary_unstyledList, // RR next to a message on the event list summary
+    .mx_RoomView_MessageList {                // RR next to a message on the messsge list
+        .mx_EventTile[data-layout=bubble] {
```
Comment (luixxiul) [inline res/css/views/right_panel/_TimelineCard.scss:147]: This block should be moved to `_EventTile.scss`. Going to address soon...

**Review by robintown — APPROVED** [review]
Sorry this took a while, but looks good! This had been bugging me for quite a while

**Comment by luixxiul** [comment]
@robintown looks like the test is broken (which seems a global issue here), would you please check whether it is irrelevant? Thanks!

**Pull request merged**